src/globals.cpp

```diff
@@ -305,6 +304,13 @@ CTweakRef<bool> enableDataSigVerifyTweak("consensus.enableDataSigVerify",
     "true if OP_DATASIGVERIFY is enabled.",
     &enableDataSigVerify);
 
+CTweak<bool> enableCanonicalTxOrder("consensus.enableCanonicalTxOrder",
+    "True if canonical transaction ordering is enabled.  Reflects the actual state so may be switched on or off by"
+    " fork time flags and blockchain reorganizations.",
+    false);
+
+std::atomic<bool> ltorEnabled{false}; // Is lexical transaction ordering enabled?
+
```

Comment:
do we still need ltorEnabled if we're using the tweak?

Reply:
seems to me that `ltorEnabled` is not used at all in this changeset.

Reply:
removed.  This was what I used before the tweak.